src/supertux/tile_set.cpp

```diff
@@ -108,4 +113,45 @@ TileSet::add_tile(int id, std::unique_ptr<Tile> tile)
   }
 }
 
+const Tile*
+TileSet::get(const uint32_t id) const
+{
+  if (id >= m_tiles.size()) {
+//    log_warning << "Invalid tile: " << id << std::endl;
+    return m_tiles[0].get();
+  } else {
+    assert(id < m_tiles.size());
+    Tile* tile = m_tiles[id].get();
+    if(tile) {
+      tile->load_images();
+      return tile;
+    } else {
+//      log_warning << "Invalid tile: " << id << std::endl;
+      return m_tiles[0].get();
+    }
+  }
+}
+
+void
+TileSet::draw_tile(DrawingContext& context, uint32_t id, const Vector& pos,
+                   int z_pos, Color color) const
+{
+  if (id == 0) return;
+  Tile* tile;
+  if (id >= m_tiles.size()) {
+    tile = NULL;
+  } else {
+    tile = m_tiles[id].get();
+  }
+
+  if (tile) {
+    tile->load_images();
+    tile->draw(context, pos, z_pos, color);
+  } else if (Editor::is_active()) { // Draw a notile sign
+    context.draw_surface(notile_surface, pos, 0, color, Blend(), z_pos);
+    context.draw_text(Resources::small_font, std::to_string(id),
```

Comment:
@Hume2 True. I noticed it because I tested using 99999 or so as tile ID which doesn't entirely fit into the tile and looks a bit ugly. Maybe in the future that could be changed, but it's good enough for now, I guess.